src/themes/oneui/base-variables/_typography.scss

```diff
@@ -1,64 +1,66 @@
 // fonts sizes
 $font-size-base: 14px !default;
-$font-size-small: .8 * $font-size-base !default;
+$font-size-small: 0.8 * $font-size-base !default;
 $font-size-normal: 1 * $font-size-base !default;
```

Comment:
If you choose to use a leading zero in decimal values (`0.8`) then it would make sense to use decimal notation everywhere (1.0).

Reply:
that was eslint, not me ;-)
But I agree, maybe better to use everywhere